soh/soh/OTRGlobals.cpp

```diff
@@ -1025,28 +1025,26 @@ extern "C" uint32_t ResourceMgr_GetGameVersion(int index) {
 uint32_t IsSceneMasterQuest(s16 sceneNum) {
     uint32_t value = 0;
     uint8_t mqMode = CVarGetInteger("gBetterDebugWarpScreenMQMode", 0);
-    if (mqMode == 1) { //non-mq wants to be mq
+    if (mqMode == 1) { // non-mq wants to be mq
         return 1;
-    } else if (mqMode == 2) {//mq wants to be non-mq
+    } else if (mqMode == 2) { // mq wants to be non-mq
```

Comment:
would be nice to use enums instead of magic numbers here but that doesn't need to block this PR